geometry.hpp

```diff
@@ -62,6 +62,10 @@ class Vector3d{
     return Vector3d(v[0]*d, v[1]*d, v[2]*d);
   }
 
+  double at( int i ) const {
+    return v[i];
+  }
+
```

Comment:
I was thinking the same thing, but waiting until I found some evidence of if/how it was used.